syntax = "proto3";

package example;

message PlayerModel{
    string name = 1;
    int32 score = 2;
}

message PlayerInputs{
    string player_name = 1;
    bool pressed_key = 2;
}